From: Paul Gortmaker <paul.gortmaker@windriver.com>

ppc4xx: use raw lock for interrupt controller

Just like every other IRQ controller, this 4xx specific chip/driver
needs to have a real non-preemptible spinlock too.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>

---
 arch/powerpc/sysdev/uic.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/sysdev/uic.c b/arch/powerpc/sysdev/uic.c
index d35405c..7add4a9 100644
--- a/arch/powerpc/sysdev/uic.c
+++ b/arch/powerpc/sysdev/uic.c
@@ -49,7 +49,7 @@ struct uic {
 	int index;
 	int dcrbase;
 
-	spinlock_t lock;
+	raw_spinlock_t lock;
 
 	/* The remapper for this UIC */
 	struct irq_host	*irqhost;
-- 
1.6.3.2

